scripts/seed-cart.sh

```
@@ -0,0 +1,62 @@
+fi
+echo "  sessionId = $SESSION_ID"
+
+# 2. 백엔드 메뉴 상위 3개 가져오기 (인기 순)
+echo "→ 인기 메뉴 3개 조회"
+MENU_IDS=$(curl -s "$BASE/api/menus/top?limit=3" \
+  | python3 -c "import json,sys;[print(m['menuId']) for m in json.load(sys.stdin)['data']]")
+
+# 3. 각 메뉴를 수량 1로 카트에 추가
+echo "→ 카트에 담기"
+for MID in $MENU_IDS; do
+  RESP=$(curl -s -X POST "$BASE/api/orders/cart/items" \
```

Shellcheck (0.11.0)

[warning] RESP appears unused. Verify use (or export if used externally).

(SC2034)